lib/actions/request_review.js

```diff
@@ -16,7 +16,7 @@ const createRequestReview = async (context, number, reviewers, actionObj) => {
       action_name: actionObj.name
     }
 
-    actionObj.log.error(JSON.stringify(errorLog))
+    actionObj.log.info(JSON.stringify(errorLog))
```

Comment:
What is the reasoning for this change?

Reply:
because we are using `info` for other known errors as well